main.css

```diff
@@ -223,3 +223,114 @@ footer {
     font-size: 0.9em;
   }
 }
+
+header {
+  background-image: url(./Assets/Images/minecraft_builds/MC_SingaporeSkyline_1.png);
+  background-repeat: none;
+  background-size: cover;
+  background-position: 50% 0%;
+  position: relative;
+}
```

Comment:
since main.css now act like global variable for some file this override some same class of its child
![image](https://user-images.githubusercontent.com/77855014/139531144-3ec12201-76fd-4553-bcbd-35971932044e.png)
> ex. this line will get override
> https://github.com/ASEAN-Build-The-Earth/AseanBTE-Website/blob/589cd56ef44afe612bf80c412ba42d06bfcc3498/Projects/Luym_Building/style.css#L44

can do !important but u gotta fix for all of main.css's chid

Reply:
I'll do the changes tomorrow